scheme/arm-cca/plugin/combined/Makefile

```
@@ -1,8 +1,8 @@
 # Copyright 2024 Contributors to the Veraison project.
 # SPDX-License-Identifier: Apache-2.0
 
-PLUGIN := ../../../bin/cca-store-handler.plugin
-GOPKG := github.com/veraison/services/scheme/cca-ssd-platform
+PLUGIN := ../../../bin/arm-cca.plugin
+GOPKG := GOPKG := github.com/veraison/services/scheme/arm-cca
```

Not sure how that managed to work!  I bow to the superior self-healing properties of GNU make :)

```suggestion
GOPKG := github.com/veraison/services/scheme/arm-cca
```

---

Ah, good catch: I shall correct this!